frontend/public/components/nav.jsx

```diff
@@ -364,10 +364,11 @@ export class Nav extends React.Component {
         <ClusterPickerNavSection />
         <div ref={this.scroller} onWheel={this.preventScroll} className="navigation-container">
           <NavSection text="Overview" icon="fa fa-tachometer" href="/overview" activePath="/overview/" onClick={this.close} />
-          <NavSection required={FLAGS.CLOUD_SERVICES} text="Applications" img={appsLogoImg} activeImg={appsLogoActiveImg} >
+
+          <NavSection required={FLAGS.CLOUD_SERVICES} text="Operators" img={operatorImg} activeImg={operatorActiveImg} >
             <ResourceNSLink model={ClusterServiceVersionModel} resource={ClusterServiceVersionModel.plural} name="Cluster Service Versions" onClick={this.close} />
             <Sep />
-            <ResourceNSLink model={CatalogSourceModel} resource={CatalogSourceModel.plural} name="Open Cloud Catalog" onClick={this.close} />
+            <ResourceNSLink model={CatalogSourceModel} resource={CatalogSourceModel.plural} name="Operator Catalog" onClick={this.close} />
```

Comment:
"Catalog Sources"

Reply:
^ @tlwu2013, @robszumski since that differs from what's in https://jira.coreos.com/browse/CONSOLE-640

Reply:
After thinking for quite a while, perhaps it'll be better for 3.11 release to make NavTitle matches with the resource name. i.e. Showing "Catalog Sources" in this case (along with "Cluster Service Versions", "Subscriptions", and "Install Plans"). 
@robszumski what do you think?

We'll have a Marketplace UI for 4.0, so this gives us more time to gauge two directions: *A.* if we'd like to blend in OLM UIs with Marketplace, or *B.* keep a clear-cut between Marketplace UI with OLM (so OLM can 100% reveal the primitive names).

Reply:
I don't think we should deviate from the actual Kubernetes resources unless we have a totally fleshed out design for something to abstract it. While it may be confusing for someone who knows _nothing_ about Operators, everything in the Console is confusing if you don't know anything about Kubernetes. I'd rather there be consistency and refer to documentation than paper over details and fail to have anyone understand the infrastructure on which they're depending.

We tried papering over OLM for Tectonic in the past and all it did was confuse users and continue to prolong the mindset that the community shouldn't understand OLM, which is quickly become the foundation for the entire platform.